src/config-schema.ts

```diff
@@ -25,8 +25,30 @@ export const configSchema = {
     _type: Type.String,
     _default: "Pharmacy",
   },
+  substitutionReason: {
```

Comment:
Note these two new configuration parameters.  Will be required, or the dropdown for these two questions will be rendered empty.